bindings/nodejs/lib/types/block/context_input/context_input.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { AccountId } from "../..";
+import { u16 } from "../../utils/type_aliases";
+
+enum ContextInputType {
+  // ToDo: commitment context input;
```

Comment:
TODO

Reply:
This is not part of this PR, there's an issue for that one, which also requires `SlotIndex`.